### PR TITLE
Added paperclip and profile pic for user

### PR DESCRIPTION
**Pull request by chaudharyprakash**

### CODE REVIEW

File: app/models/picture.rb [hunk 1]
```diff
@@ -1,3 +1,6 @@
 class Picture < ApplicationRecord
 	belongs_to :imageable, polymorphic: true
+
```
Comment (charusat09) [inline app/models/picture.rb:3]: @chaudharyprakash again will you please remove this extra space ?

File: app/views/users/registrations/new.html.erb [hunk 2]
```diff
@@ -24,6 +24,13 @@
   </div>
 
   <div class="field">
+    <%= f.fields_for :picture, resource.picture do |picture_fields| %>
+      <%= picture_fields.label :profile_picture %><br />
+      <%= picture_fields.file_field :image,accept: 'image/png,image/gif,image/jpeg'%>
```
Comment (charusat09) [inline app/views/users/registrations/new.html.erb]: @chaudharyprakash @hardikupadhyay16 I think this validation is handled in model, isn't it ??

Comment (hardikupadhyay16) [reply app/views/users/registrations/new.html.erb]: @charusat09 Yes, we have handled validation in model but I also set front end validation so no one can play with it.

Comment (charusat09) [reply app/views/users/registrations/new.html.erb]: @hardikupadhyay16 that's great... so for that you can use helper class. Put this in helper and call that method here i.e. image_format_validation

Comment (hardikupadhyay16) [reply app/views/users/registrations/new.html.erb]: @charusat09 Sure, Thanks for correction.

Comment (charusat09) [reply app/views/users/registrations/new.html.erb]: you are welcome @hardikupadhyay16

**Pull request merged**